pages/user/index.vue

```diff
@@ -6,8 +6,29 @@
 
   const authStore = useAuthStore()
   const { user } = storeToRefs(authStore)
+
+  const localePath = useLocalePath()
+  const goToEdit = () => {
+    // asume normal user
+    navigateTo(localePath(`/user/edit/${user?.id}`))
+  }
 </script>
 
 <template>
-  <div>Welcome {{ user?.name }}!</div>
+  <div>
+    <div>Welcome {{ user?.name }}!</div>
+    <!--Edit button-->
+    <VBtn
+      size="large"
+      variant="text"
+      :ripple="false"
+      :class="`rounded-lg d-flex justify-start px-3 text-background
+        ${'text-background'}`"
+      :width="'45%'"
+      :active="true"
+      @click="goToEdit"
+    >
+      Edit Account
```

Comment:
I wound add internationalisation to this, as any user should be able to visit this. Maybe replace this with i.e. `{{ $t('edit.event.create.action') }}`. Check out what I've done with `EditForm.vue` and `locales/*` in #146 if my comments aren't clear :)